code/modules/projectiles/guns/ballistic.dm

```diff
@@ -301,13 +322,30 @@
 	if (can_be_sawn_off)
 		if (sawoff(user, A))
 			return
+	
+	if(can_misfire && istype(A, /obj/item/stack/sheet/cloth))
```

Comment:
you can't use a rag or gauze for this?

Reply:
Let's just say _its more annoying to get to work_ and leave it at that.